pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -328,9 +332,14 @@ func (p *PolicyData) EnsureRBACPolicy() genericapiserver.PostStartHookFunc {
 					if err != nil {
 						// don't fail on failures, try to create as many as you can
 						utilruntime.HandleError(fmt.Errorf("unable to reconcile rolebinding.%s/%s in %v: %v", rbac.GroupName, roleBinding.Name, namespace, err))
+						failedReconciliation = true
 					}
 				}
 			}
+			// failed to reconcile some objects, retry
+			if failedReconciliation {
```

Comment:
if we are consider about changing the no-retry behavior, we can put this under the StorageVersionAPI feature gate, just to deflake the tests. 

but according to https://github.com/kubernetes/kubernetes/blob/667d1c2c3fe7f6cece4f74dfd28983d5bdbf4208/pkg/registry/rbac/rest/storage_rbac.go#L174-L175, we should retry and reconcile these objects properly.